README.md

```diff
@@ -349,6 +349,42 @@ expcet(page).to be_accessible.within('.main').checking_only :document-title, :la
 expect(page).to be_accessible.according_to(:best-practice).checking(:aria-roles, :definition-list)
 ```
 
+# MiniTest
+
+## Be Accessible Matcher
+
+`axe/matchers` provides a new matcher `BeAccessible` as a class to assist in validating the accessibility of a page using a web driver.
+
+## Configuration
+
+Require the matchers in your test file.
+
+``` ruby
+require 'axe/matchers'
+```
+
+## Built-Int Accessibility Checks
+
+To construct an axe accessibility check, begin with `Axe::Matchers.be_accessible`, and append any clauses necessary. Then call `matches?` with the page.
+
+``` ruby
+Axe::Matchers.be_accessible.matches? page
```

Comment:
Let's rip the minitest stuff out (of the docs) for now and attack that separately.